extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -236,6 +237,7 @@ private function handleBlockPost() {
 			'author_id'  => $this->wg->User->getId(),
 			'reason'     => $this->wg->Request->getText( 'wpPhalanxReason' ),
 			'comment'    => $this->wg->Request->getText( 'wpPhalanxComment' ),
+			'lang'       => $this->wg->Request->getVal( 'wpPhalanxLanguages', null ),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)